ILSpy/Themes/ThemeManager.cs

```diff
@@ -58,6 +59,8 @@ private ThemeManager()
 			set => UpdateTheme(value);
 		}
 
+		public bool ThemeIsDark => IsDarkTheme(Theme);
```

Comment:
What is this property used for?

Reply:
I just created this property, so if anyone needs it, they can use it directly.

Reply:
Of course, if you think it's not necessary, you can also delete it.